README.md

```diff
@@ -1,24 +1,24 @@
-@asymmetrik/fhir-server-mongo
-=============================
+`@asymmetrik/node-fhir-server-mongo` [![Build Status](https://travis-ci.org/Asymmetrik/node-fhir-server-mongo.svg?branch=develop)](https://travis-ci.org/Asymmetrik/node-fhir-server-mongo)
+====================================
 
 ## Intro
 
 ## Dependencies
-> This project requires nodejs and mongo to be installed on your system.
+> This project requires docker for local development. You can deploy it with or without docker. To deploy it separately, you need a server with the latest LTS for [Node.js](https://nodejs.org/en/) installed and a mongo database that can be reached by the node server.
```

Comment:
Is Docker required for local dev, or is it only a convenience thing?  I'd think that as long as you setup your environment vars correctly you should be able to run it locally without Docker.